source/_integrations/rotarex.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: "Rotarex DIMES SRG"
+description: "Instructions on how to integrate your Rotarex gas tanks into Home Assistant."
+documentation: https://rotarexsrg.com/category/digital-measurement-system
+ha_release: 2026.1
+ha_domain: rotarex
+ha_iot_class: cloud_polling
+ha_codeowners:
+  - '@Rotarex'
+---
+
+The Rotarex DIMES SRG {% term integration %} allows you to monitor the level, battery, and last sync time of your Rotarex DIMES SRG Camping gas tanks in Home Assistant.
+
+This integration uses the official Rotarex cloud API to fetch the latest data.
+
+## Configuration
+
+To add the **Rotarex DIMES SRG** integration to your Home Assistant instance, use this My button:
+
+[![Open your Home Assistant instance and start setting up a new integration.](https://my.home-assistant.io/badges/config_flow_start.svg)](https://my.home-assistant.io/redirect/config_flow_start/?domain=rotarex)
```

Comment:
```suggestion
{% include integrations/config_flow.md %}
```
We have this [reusable text block](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#reusable-text) you can use for this. See also https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L51.